### ISSUE TITLE
Polarr extension not available

### ISSUE THREAD

**Issue by ruiseabra** [Polarrco/polarr_mac_support#3]
After updating to the latest version of El Capitan the Polarr extension is not available anymore.
<img width="651" alt="screen shot 2016-05-24 at 13 13 12" src="https://cloud.githubusercontent.com/assets/6738826/15505052/59621b28-21b1-11e6-9917-d6a547cb1931.png">

I have made sure I had the latest Polarr version (1.4.0) but it is still not showing.
<img width="475" alt="screen shot 2016-05-24 at 13 12 42" src="https://cloud.githubusercontent.com/assets/6738826/15505050/58324872-21b1-11e6-8c15-989114da3e55.png">

Is this related to the update of El Capitain? Is it going to be fixed?
Btw, Polarr rocks.

**Comment by bwang29** [comment]
This is super strange. Does restarting the computer help?

**Comment by ruiseabra** [comment]
Nope, still no photos extensions despite a couple of reboots... and I've also tried to delete and re-install Polarr. No luck
